python/lsst/ap/association/trailedSourceFilter.py

```diff
@@ -0,0 +1,110 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""A simple implementation of source association task for ap_verify.
+"""
+
+__all__ = ["TrailedSourceFilterTask", "TrailedSourceFilterConfig"]
+
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.utils.timer import timeMethod
+
+# import numpy as np
+import pandas as pd
+
+# Enforce an error for unsafe column/array value setting in pandas.
+pd.options.mode.chained_assignment = 'raise'
```

Comment:
I followed what was being done in association.py. If it applies to all code run after, then since this runs after association.py has imported and run it would apply to the trailedSourceFilter as well. However, with what you said, should we reconsider it being set in association.py? What would be a better way to set it?

Reply:
I've filed [DM-40769](https://jira.lsstcorp.org/browse/DM-40769) to explore this question. We probably should be doing this inside a context manager in `run`, if we actually want to do it at all. Don't worry about it here; we'll sort it out on that ticket.